IntegrationTests/IRVMR/script/compileHLS.tcl

```diff
@@ -1,28 +1,19 @@
 # Common settings
 source ../../../project/env_hls.tcl
+file link -symbolic common ../../common
+file link -symbolic MemPrints ../../../emData/MemPrints
+source common/script/build_ip.tcl
 set CFLAGS {-std=c++11 -I../../../TrackletAlgorithm -I../../../TopFunctions}
 set cwd [pwd]
 
 # Build IR_DTC_PS10G_3_A
 open_project -reset inputrouter
```

Comment:
It would be nice to provide option (normally disabled to save CPU) to run also csim & cosim, to check that the HLS module gives correct results. And to throw an error if they reveal problems ...

Reply:
This would require a little more work. In particular, we need a way of switching the top function that is called during cosimulation. Currently, this is hard-coded in most of the C++ test benches. Also, all the test benches are configured to use the top functions generated from the full configuration, and we need to figure out how to switch to those generated from the reduced configuration from inside the Tcl script.

I would propose addressing this in a followup PR after this one is merged, because I agree that it would be very nice if this were possible.